src/server/hook_func.c

```diff
@@ -3676,6 +3684,95 @@ write_hook_accept_debug_output_and_close(void)
 	}
 }
 
+/**
+ * @brief
+ * 		getting the vnode attributes and resource list
+ *			each node attribute will be of the format:
+ *			<node_name>.<attr_name>
+ *
+ * @see
+ * 		run_periodic_hook
+ *
+ * @return void
+ */
+static pbs_list_head *
+get_vnode_list(void){
+	int i;
+	int index;
+	char name_str_buf[STRBUF + 1] = {'\0'};
```

Comment:
STRBUF is set to LOG_BUF_SIZE which is of size 4096. I felt this is more than enough space to hold the string. No?

Reply:
Well you are right... But Ideally, it should be associated with node name length. If in future anything changes in maxnodename or we decrease log_buf_size then it may cause problems. what do you think?

Reply:
I feel we would always maintain a larger value for log_buf_size and even if we change maxnodename it would be lesser than log_buf_size. Another issue here is how to calculate exact size to store attribute name?